.github/workflows/bin/spack-reviewers.py

```diff
@@ -47,6 +48,11 @@ def main():
 
     added_reviewers = reviewers - existing_reviewers
     if added_reviewers:
+        # exit early if the PR would notify a very large number of people
+        if len(added_reviewers) > NUM_MAX_REVIEWERS:
+            print(f"[PR #{pr_number}]: skipping PR that would ping >{NUM_MAX_REVIEWERS} people")
```

Comment:
I think this should post a comment along the lines of

```
Your pull request exceeds the package change limit ({NUM_MAX_REVIEWERS}). Please consider breaking this up into smaller PRs.
```

And then request review from `@spack-releasers` or something similar since that is a group that can merge with bypass for cases like license header changes or large deprecation/removal situations.